azure/resource-containers/pf-management-group/functions/public/save-pfmanagementgroup.ps1

```diff
@@ -0,0 +1,14 @@
+Function Save-PfManagementGroup{
+    Param (
+        [Parameter(Mandatory = $true)] 
+        [PfManagementGroupDefinition]$Definition
+    )
+
+    Write-Verbose("Resolving dependencies for $($Definition.Options.Name) ...")
+    Resolve-PfManagementGroup -Definition $Definition
```

Comment:
This is where we can resolve any dependencies now or to start building our deployment plan